templates/ChannelPricingLogEntry/Grid/Field/originalPrice.html.twig

```diff
@@ -0,0 +1,10 @@
+{% import "@SyliusAdmin/Common/Macro/money.html.twig" as money %}
```

Comment:
Either this file should be within the `Admin` directory or a different money macro should be used, the same case with the `price.html.twig` file.